### ISSUE TITLE
SpecialZoomLevel.PageFit Race Condition that causes the document to default to -3%

### ISSUE THREAD

**Issue by cabal-daniel** [react-pdf-viewer/react-pdf-viewer#597]
Our s3 remote documents sometimes take a while to load (couple of seconds). Sometimes this causes weird behaviors in the viewers that cause the document to default to -3% zoom when it appears. It's occurrence varies based on how fast the document loads so it doesn't always happen.

This happens on
```
<Viewer
   defaultScale={SpecialZoomLevel.PageFit}
```

We remedied it by setting this to a hard scale level with

```
<Viewer
   defaultScale={1.2}
````

It's not a super annoying bug but it makes the really cool `PageFit` feature unusable for us. Just wanted to let you be aware of this. 

How it appears
![Screen Shot 2021-06-06 at 8 40 11 PM](https://user-images.githubusercontent.com/65152299/120956186-d65c3e00-c707-11eb-95a4-e7535d714890.png)

What the document is supposed to look like after we adjust the zoom
![Screen Shot 2021-06-06 at 8 40 22 PM](https://user-images.githubusercontent.com/65152299/120956208-e1af6980-c707-11eb-8891-41182a31078b.png)

Really love this project! Bought a license several months back!

**Comment by phuoc-ng** [comment]
> Really love this project! Bought a license several months back!

Thanks for supporting my work.

Regarding the issue, can you let me know the version of package you're using? If you don't use the latest version, can you upgrade to the latest one?

**Comment by cabal-daniel** [comment]
I am on the latest 2.5.0

**Comment by phuoc-ng** [comment]
@cabal-daniel I think it's similar to #698 and both issues should be fixed by the PR #728.
The fix will be included in the next 2.7.2.

I will let you know when I release 2.7.2, so you can update.

**Comment by phuoc-ng** [comment]
@cabal-daniel 
[v2.7.2 is released](https://react-pdf-viewer.dev/updates/). Please upgrade.
Let me know if the issue is still there.
